charts/internal/shoot-system-components/charts/rook-ceph/values.yaml

```diff
@@ -0,0 +1,2 @@
+images:
+   rook-ceph: image-repository:image-tag
```

Comment:
You might want to add the other images here

Reply:
Done, I think.